src/styles/_catalog.scss

```diff
@@ -1,22 +1,36 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 0 48px;
```

Comment:
The gap between cards should only affect the horizontal gap, but it currently affects both horizontal and vertical gaps. Use 'column-gap' instead of 'gap' to fix this issue.